Tests/Fixture.swift

```diff
@@ -40,7 +40,7 @@ struct Fixture {
     
     /// The URL of the fixture within the test bundle.
     var URL: NSURL {
-        let bundle = NSBundle(identifier: "com.diephouse.matt.TentacleTests")!
+        let bundle = NSBundle(forClass: GitHubErrorTests.self)
```

Comment:
This file is used in 2 targets: `TentacleTests` and `update-test-fixtures`. Because of the latter, it can't use one of the `XCTestCase` classes to get the bundle. Maybe a better idea would be to add a private class to this file and use that here?